Library/Formula/dwarfutils.rb

```diff
@@ -0,0 +1,28 @@
+class Dwarfutils < Formula
+  desc "lib and utility to dump and produce DWARF debug info in ELF objects"
+  homepage "http://www.prevanders.net/dwarf.html"
+  url "http://www.prevanders.net/libdwarf-20150507.tar.gz"
+  sha256 "29aa8d07db659d7d7af7079854cf42c09bf74d303942159cbfee82d655549870"
+
+  depends_on "libelf" => :build
+
+  def install
+    # Disable building the shared library, since the package does not understand
+    # OS X naming quirks (dylib extension) and rpath quirks, and the static
+    # library works just fine.
+    system "./configure", "--disable-shared"
+    system "make"
+
+    # The package doesn't contain a "make install" target, as documented in its
+    # README. Just follow the directions the readme suggests.
+    bin.install "dwarfdump/dwarfdump"
+    man1.install "dwarfdump/dwarfdump.1"
+    lib.install "libdwarf/libdwarf.a"
+    include.install "libdwarf/dwarf.h"
+    include.install "libdwarf/libdwarf.h"
+  end
+
+  test do
+    system "#{bin}/dwarfdump", "-V"
```

Comment:
An extensive test is pretty difficult, since this package is designed to work on ELF objects, which are harder to come by on OS X unless you're doing something fairly weird/complex. A somewhat better test might be a simple C program that linked against libdwarf and called some sort of init function, which I will update this diff to add.